main.go

```diff
@@ -46,11 +47,27 @@ func main() {
 	}
 
 	repl := replicator.New(*cfg)
+
 	if *generateChDDL {
 		if err := repl.GenerateChDDL(); err != nil {
 			fmt.Fprintf(os.Stderr, "could not create tables on the clickhouse side: %v\n", err)
 			os.Exit(1)
 		}
+	} else if *onlySync {
+		var err error
+
+		if err = repl.Init(); err == nil {
+			var syncTables []config.PgTableName
```

Comment:
instead of declaring var I'd do the following:
```
if syncTables, err := repl.GetSyncTables(); err == nil {
```
as you don't use that var anywhere else